pages/common/git-commit.md

```diff
@@ -15,6 +15,10 @@
 
 `git commit -a -m "{{message}}"`
 
+- Commit staged files and sign them with the GPG key defined in the config:
```

Comment:
```suggestion
- Commit staged files and sign them with the GPG key defined in .gitconfig:
```

Reply:
What about Mac and Windows users?

Reply:
Ok, what do they have though?

Reply:
Nevermind, they the file is called `.gitconfig` everywhere. Since Mac also supports it how about if we even used `~/.gitconfig`?

Reply:
Does windows use ~ ?

Then again who is using git for windows anyways?
We could leave it as what you have suggested, or help Linux and Mac users in locating the file for git configuration.

If we are not going to support windows here then it should be ~/.gitconfig

As you have mentioned

Reply:
`git-config` page uses `(stored in `~/.gitconfig`)`